src/Microsoft.ML.Data/Transforms/ExtensionsCatalog.cs

```diff
@@ -85,8 +85,8 @@ public static ColumnSelectingEstimator DropColumns(this TransformsCatalog catalo
         public static ColumnSelectingEstimator SelectColumns(this TransformsCatalog catalog,
```

Comment:
>SelectColumns [](start = 47, length = 13)

can we remove this altogether? I don't see a value for this. Actually, I would rename `KeepColumns` to `SelectColumns`

Reply:
If they want to drop some, and keep some in one shot? Also I was following the constructors in the class; if de decide to remove Keep, shall it happen there, and as a separate work item?

---
In reply to: [233621944](https://github.com/dotnet/machinelearning/pull/1606#discussion_r233621944) [](ancestors = 233621944)